test/message-generator/test/standard-coverage-test/standard-coverage-test.json

```diff
@@ -181,27 +181,6 @@
                 }
             }
         },
-        {
-            "command": "cargo",
-            "args": [
-                        "run",
-                        "../../../test/message-generator/mock/template-provider-mock1.json"
-            ],
-            "conditions": {
-                "WithConditions": {
-                    "conditions": [
-                        {
-                            "output_string": "Running `target/debug/message_generator_sv2 ../../../test/message-generator/mock/template-provider-mock1.json`",
-                            "output_location": "StdErr",
-                            "late_condition": false,
-                            "condition": true
-                        }
-                    ],
-                    "timer_secs": 320,
-                    "warn_no_panic": false
-                }
-            }
-        },
```

Comment:
why is this being removed?

Reply:
because
1. the TP mock is launched in the same command above
2. there is no file at that address